Tools/bin/efficiency.cc

```diff
@@ -11,6 +11,7 @@
 #include "TGraphAsymmErrors.h"
 #include "TH1.h"
 #include "TLatex.h"
+#include <TMath.h>
```

Comment:
These changes conflict with another commit in the `process-2018` branch that implements the same changes. You can fix this either in the conflict resolution web GUI or by directly copying the [existing file in the repository](https://raw.githubusercontent.com/cms-exotica-diphotons/diphoton-analysis/process-2018/Tools/bin/efficiency.cc) and committing that file to your `process-2018` branch.

Reply:
The syntax `#include "TMath.h"` should be preferred here. Angle brackets are generally only used for system headers. See the [gcc documentation](https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html) for details.

This does need to be resolved; the pull request cannot be merged until conflicts are resolved.

Reply:
I used the preferred syntax.